protos/dcs/net/v0/net.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+package dcs.net.v0;
+import "dcs/common/v0/common.proto";
+
+service NetService {
+  // https://wiki.hoggitworld.com/view/DCS_func_send_chat_to
+  rpc SendChatTo(SendChatToRequest) returns (SendChatToResponse) {}
+
+  // https://wiki.hoggitworld.com/view/DCS_func_send_chat
+  rpc SendChat(SendChatRequest) returns (SendChatResponse) {}
+}
+
+message SendChatToRequest {
+  string message = 1;
+  uint32 to_player_id = 2;
+  optional uint32 from_player_id = 3;
```

Comment:
I am kind of iffy about this, not the fact we added it here, but the whole idea of spoofing another player's ID being part of the API. That just seems like something that is abusable. I know that originally it could only be called in a trusted setting and DCS-gRPC clients should _still_ be trusted but I am still uneasy about it.

Can we think of any situations where we acually could use this?

Reply:
I understand your concern of a message being impersonated by another player, and the DCS-gRPC service facilitating that vector. Consequences of emulating social abuse is far more damaging to the community than "system abuse" where cheaters and hackers use to gain an advantage is repairable.

When I was writing that line of code, I was only thinking about the clients/DCS-gRPC/server being threat-free from external malicious code; not considering the severity of the breach.

Given that I cannot think of a good motivation to mimic the `fromPlayerId` parameter (where it supersedes alternative means), I will redact these parameters with a comment explaining the reasons for the redaction. If someone comes up with a good use case, they are able to present their case.

Reply:
Done in: https://github.com/DCS-gRPC/rust-server/pull/94/commits/94f8307fa34c4e79a10d359fe57b1863453eaee0